pkgs/build-support/rust/build-rust-package/default.nix

```diff
@@ -154,7 +155,28 @@ stdenv.mkDerivation ((removeAttrs args [ "depsExtraArgs" "cargoUpdateHook" "carg
 
   strictDeps = true;
 
-  passthru = { inherit cargoDeps; } // (args.passthru or {});
+  passthru = {
+    inherit cargoDeps;
+    sbom = stdenv.mkDerivation {
+      name = "${name}-sbom.json";
```

Comment:
```suggestion
      pname = "${pname}-sbom.json";
      inherit version;

```

Reply:
Is `pname` really required, as there's no version?

Reply:
reusing the version from the derivation itself makes sense I think.

Reply:
Preferably, we'd do `name = "${pname}-{version}.json"`, to keep the extension.

Reply:
According to the contributing guide, every package should use pname+version.

Reply:
This isn't a package, though, it doesn't make sense in this context.